openedx_lti_tool_plugin/resource_link_launch/ags/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Test ags module."""
+
+MODULE_PATH = 'openedx_lti_tool_plugin.resource_link_launch.ags'
```

Comment:
@kuipumu You are setting this MODULE_PATH in each testing module. What do you think move it into the constants module or in an more general __init__.py as a dict to obtain the the desire path calling it dfor example `MODULE_PATH.get('ags_path')` (It is just an example.)

Reply:
@anfbermudezme I think for now I will keep it by setting a MODULE_PATH on each module that requires it because I'm not sure right now what could be the best approach to improve this and also this PR main focus was on relocating the resource link launch code, improving the tests or code quality was an extra and could be improved in another iteration, I will think of another approach to this based on your suggestion so we can apply it to new and existing tests on a future ticket